### ISSUE TITLE
Ingestion call failed with error: Could not establish secure connection

### ISSUE THREAD

**Issue by MatkovIvan** [microsoft/appcenter-sdk-react-native#663]
### **Description**

The logs do not appear in the AppCenter dashboard.
It's moved from https://github.com/microsoft/appcenter-sdk-react-native/issues/656#issuecomment-518277764. See details there.

### **Details**

- Platform: iOS
- Logs:
```
TIC SSL Trust Error [8:0x6000013e5380]: 3:0
NSURLSession/NSURLConnection HTTP load failed (kCFStreamErrorDomainSSL, -9813)
Task <341AA53A-E549-4E7E-A5CB-72ED638F677C>.<1> HTTP load failed (error code: -1202 [3:-9813])
Task <341AA53A-E549-4E7E-A5CB-72ED638F677C>.<1> finished with error - code: -1202
[AppCenter] DEBUG: -[MSHttpIngestion sendCallAsync:]_block_invoke/233 HTTP request error with code: -1202, domain: NSURLErrorDomain, description: The certificate for this server is invalid. You might be connecting to a server that is pretending to be “in.appcenter.ms” which could put your confidential information at risk.
[AppCenter] INFO: -[MSIngestionCall ingestion:callCompletedWithResponse:data:error:]/86 Ingestion call failed with error: Could not establish secure connection.
```

**Comment by MatkovIvan** [comment]
@Narcis11
1. Did you modify the list of root certificates somehow?
2. Is the date/time correct on the device?
3. Do you use any proxies?

**Comment by Narcis11** [comment]
@MatkovIvan , to your questions:

1. No, I did not.
2. I think so, yes.
3. Initially, when I was debugging in issue https://github.com/microsoft/appcenter-sdk-react-native/issues/656 , I was in a corporate network. I tried a few minutes ago in my home network and it worked (events are sent). Note that I did have to add the line 
`[MSAppCenter start:@"4c1f511e-385b-4e18-b23e-1e1be15a6f8a" withServices:@[[MSAnalytics class]]];`
in my AppDelegate.m file in order for the analytics events to be sent, which is a bit strange. 

Anyways, it works now, so the issue can be closed. Thanks a lot for the help, much appreciated! 🙏🙏🙏🙏
